service/service.go

```diff
@@ -153,3 +132,40 @@ func (srv *service) Shutdown(ctx context.Context) error {
 	// TODO: Shutdown TracerProvider, MeterProvider, and Sync Logger.
 	return errs
 }
+
+func (srv *service) initPipeline(set *settings) error {
+	var err error
```

Comment:
nit: Seems kind of unusual to declare this outside of any individual condition, since each error is returned immediately.

Reply:
This was copying the pattern that was in `newService` before I refactored it. This is done because in the assignments if there is no error it assigns to an already existing field on the `srv` object. I think the alternative would be to create tmp variables to assign to the `srv` fields but I don't think that's necessarily cleaner.